boards/DIM/Inc/App/states/App_DriveState.h

```diff
@@ -23,14 +23,59 @@ const struct State *App_GetDriveState(void);
 
 /**
  * Check CAN bus for BMS Fault messages
- * @param can_rx Dim CAN RX interface for recieving messages
  * @return True if any BMS Fault is present
  */
-bool App_DriveState_HasBmsFault(void);
+bool App_DriveState_HasBMSFault(void);
+
+/**
+ * Check CAN bus for DCM Fault messages
+ * @return True if any DCM Fault is present
+ */
+bool App_DriveState_HasDCMFault(void);
+
+/**
+ * Check CAN bus for DIM Fault messages
+ * @return True if any DIM Fault is present
+ */
+bool App_DriveState_HasDIMFault(void);
+
+/**
+ * Check CAN bus for FSM Fault messages
+ * @return True if any FSM Fault is present
+ */
+bool App_DriveState_HasFSMFault(void);
+
+/**
+ * Check CAN bus for PDM Fault messages
+ * @return True if any PDM Fault is present
+ */
+bool App_DriveState_HasPDMFault(void);
```

Comment:
I think it's bad practice to declare functions in headers that aren't used in any other files. Better to declare them as static at the top of the c file

Reply:
Sounds good